docs/gravitino-server-config.md

```diff
@@ -84,20 +84,32 @@ Refer to [security](security.md) for HTTPS and authentication configurations.
 
 There are three types of catalog properties:
 
-1. **Gravitino-defined properties**: These properties simplify the catalog creation process.
-2. **Properties with the `gravitino.bypass.` prefix**: These properties aren't managed by Gravitino; instead, they bypass the underlying system for advanced usage.
-3. **Other properties**: Stored in Gravitino storage, these properties don't bypass the underlying system.
+1. **Gravitino defined properties**: Gravitino defines these properties as the necessary
+   configurations for the catalog to work properly.
+2. **Properties with the `gravitino.bypass.` prefix**: These properties are not managed by
+   Gravitino and pass directly to the underlying system for advanced usage.
+3. **Other properties**: Gravitino doesn't leverage these properties, just store them. Users
+   can use them for their own purposes.
 
-Catalog properties are either defined in catalog configuration files as default values or specified explicitly when creating a catalog.
+Catalog properties are either defined in catalog configuration files as default values or
+specified as `properties` explicitly when creating a catalog.
 
 :::info
-Explicit specifications take precedence over formal configurations.
-:::
+The catalog properties explicitly specified in the `properties` field take precedence over the
+default values in the catalog configuration file.
 
-:::caution
 These rules only apply to the catalog properties and don't affect the schema or table properties.
 :::
 
+Below is a list of catalog properties that will be used all Gravitino catalogs:
+
+| Configuration item | Description                                                                                                         | Default value | Required | Since version |
+|--------------------|---------------------------------------------------------------------------------------------------------------------|---------------|----------|---------------|
+| `package`          | The path of the catalog package. Gravitino leverages this path to load the related catalog libs and configurations. | (none)        | No       | 0.5.0         |
```

Comment:
I think we should add some information about the right contents in the package like jars are under the sub-directory `libs` and configuration files lies in `conf`.

Reply:
+1